Off-Top-App/off-top-python
# OT-SplitDomainService | Moved routes from app.py to files grouped by their route uses.

### Task Completed

- moved routes from app.y to files grouped by their route uses.
- Two Files called UserReportsService.py & UserSessionService.py
- All routes relating to reports in UserReportsService.py
- All routes relating to session in UserSesssionService.py
- Renamed app.py to server.py

### How to test
- navigate to /off-top-python/backend-services
- run `python server.py`
- open postman
- use `localhost:5000/<route-name>` to test and make sure you don't have any issues with these in postman. Let me know if you get any issues. `<route-name>` can be seen below.

- route-name list:
  - merge-user-data
  - get-user-session-duration
  - get-user-topic
  - get-users-avg-scores
  - get-users-profession-with-score
  - get-user-information
  - get-all-sessions
  - insert-session
